Makefile

```diff
@@ -42,7 +42,6 @@ run-end-to-end-test:
 stop-end-to-end-test:
 	docker-compose down
 
-include docker/build.mk
```

Comment:
Why is this removed now?

Reply:
That file basically just has base images build steps and the project-builder image. Both of these are generated using the images yaml file.

Reply:
Is anyone using this file anymore? Can we delete it?

Reply:
Will delete it. I'll put the todo from that file into `images_types.yaml`.